.travis.yml

```diff
@@ -33,10 +34,13 @@ before_install:
       brew install yajl;
       brew install augeas --HEAD;
       brew install dbus;
-      brew install qt5;
+      # Building Qt5 currently takes too much time.
```

Comment:
> Why are you trying to build qt5, aren't there any binaries (with dbus)?

It looks like the current binary version of the Homebrew formula (a.k.a. ”bottle”) does not include D-Bus support. 

The [Qt download page](https://www.qt.io/download-open-source) lists disk image most likely containing binary versions of Qt. The current disk image is quite large though (about 800MiB). It will take me some time to download the image. After  the download finishes I will check if we can use the disk image to install Qt in the Travis build.

> If they do not exist, we should simply opt-out dbus…

Not requiring D-Bus support when building the Qt-GUI sounds like a good idea.

> Btw. is it possible to build mac os X packages on travis: `.dmg`, homebrew or something similar?

In theory [yes](http://sleepycoders.blogspot.co.at/2013/03/sharing-travis-ci-generated-files.html), but as far as I know there exists no official support for something like that.